src/main/java/edu/hawaii/its/groupings/controller/ErrorRestController.java

```diff
@@ -29,4 +45,14 @@ public ResponseEntity<Void> feedbackError(@RequestBody Map<String, String> body,
         session.setAttribute("feedback", new Feedback(exceptionMessage));
         return new ResponseEntity<>(HttpStatus.NO_CONTENT);
     }
+
+    @PreAuthorize("hasRole('ADMIN')")
+    @GetMapping(value = "/api/groupings/testing/exception")
+    public ResponseEntity<String> throwException() {
+        logger.info("Entered REST throwException...");
+
+        String currentUid = userContextService.getCurrentUid();
+        String uri = String.format(API_2_1_BASE + "/exception");
+        return httpRequestService.makeApiRequest(currentUid, uri, HttpMethod.GET);
+    }
```

Comment:
Does calling `throw new Exeception()` work here instead of making a 404 API request?

Reply:
I think calling throw new RuntimeException() might work, would you like to change it?

Reply:
There's a ExceptionForTesting class that the API has. Could you move it to the UI and make sure to delete it from the API?